doc/news/changes/minor/20161223DenisDavydov

```diff
@@ -0,0 +1,4 @@
+New: Add IndexSet::index_within_set (const size_type global_index, size_type &local_index)
```

Comment:
doxygen is going to render this in a funny way, and will probably not link to the correct function. I typically write this as "Add a variant of IndexSet::index_within_set() that takes two arguments and that does X, Y, Z."